public/pages/AnomalyCharts/containers/__tests__/AnomaliesChart.test.tsx

```diff
@@ -74,6 +74,6 @@ describe('<AnomaliesChart /> spec', () => {
   test('renders the component', () => {
     console.error = jest.fn();
     const { getByText } = renderDataFilter();
-    expect(getByText('anomaly grade')).not.toBeNull();
+    expect(getByText('Sample anomaly grade')).not.toBeNull();
```

Comment:
Where is 'Sample anomaly grade' rendered on anomalies chart page?

Reply:
https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/blob/master/public/pages/AnomalyCharts/containers/AnomaliesChart.tsx#L280